src/components/Arguments/index.tsx

```diff
@@ -411,15 +410,17 @@ const Arguments: React.FC<ArgumentsProps> = (props) => {
             </>
           )}
 
-          <ErrorsList list={problems.error} actions={actions} />
+          {/*<ErrorsList list={problems.error} actions={actions} />*/}
```

Comment:
same as line 360

Reply:
leaving it commented for potential future work